cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
There a few files like this that Cypress auto-generates every time it runs. I've checked them in so you don't end up with files outside of version control that will just get recreated if you try to delete them.

Reply:
Looks like we can [disable support files](https://docs.cypress.io/guides/references/configuration#Folders-Files) if we want but they don't seem to be doing any harm.

Reply:
I'm going to disable them, I think its preferable to having these hanging around